README.org

```diff
@@ -57,6 +57,34 @@ whether a directory is the project root:
   (setq pytest-project-root-test (lambda (dirname) (equal dirname "foo")))
 #+END_SRC
 
+By default, test paths passed to pytest are absolute paths:
+#+BEGIN_SRC sh
+  cd /project/root && pytest /base/path/relative/path/to/test.py
```

Comment:
This should read `cd /project/root && py.test '/project/root/path/to/test.py'`. I think this will help explain the use case more clearly.